website/docs/r/lightsail_disk.html.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: "aws"
+page_title: "AWS: aws_lightsail_disk"
+sidebar_current: "docs-aws-resource-lightsail-disk"
+description: |-
+  Provides a Lightsail Disk
+---
+
+# Resource: aws_lightsail_instance
+
+Provides a Lightsail Disk. Amazon Lightsail is a service to provide easy virtual private servers
+with custom software already setup. See [What is Amazon Lightsail?](https://lightsail.aws.amazon.com/ls/docs/getting-started/article/what-is-amazon-lightsail)
+for more information.
+
+~> **Note:** Lightsail is currently only supported in a limited number of AWS Regions, please see ["Regions and Availability Zones in Amazon Lightsail"](https://lightsail.aws.amazon.com/ls/docs/overview/article/understanding-regions-and-availability-zones-in-amazon-lightsail) for more details
+
+## Example Usage
+
+```hcl
+# Create a new GitLab Lightsail Instance
```

Comment:
This comment refers to `instance` rather than `disk`

Reply:
You're right. Fixed.